### DISCUSSION TITLE
Roadmap for tanstack/router 1.0.0

### DISCUSSION THREAD

**Discussion by valerii15298** [TanStack/router#565] [Q&A] [answered]
First of all, thank you for typescript first router library for react. I did not use it yet, but from examples and docs it already seems very promising and ground breaking for me compared to react-router.

I would like to use it for my company projects and personal ones.

Since we didn't reach 1.0.0 version, it would be very helpful to understand what features/public APIs will change, or at least some roadmap. So that we can understand what risk do we take using it in production projects.

I wanna use it, but "beta" label doesn't give me enough understanding of risk which will be using it. How ready is it for prod?

**Comment by sagaban** [comment]
Is there any feedback on this?

**Comment by stychu** [comment]
@tannerlinsley would you mind giving some light on this? Im also interested for production use and recently saw the promotion video for the router.
Loved the video btw! It's sick! Keep up good work 👍

**Comment by tannerlinsley** [comment, marked as answer]
Nearing 1.0. Just fixing bugs and performance concerns.

**Reply by valerii15298** [reply]
Thank you, Tanner, for your hard work!!!!!!!!!!!!!!!!
